pkg/keys/constants.go

```diff
@@ -460,6 +460,7 @@ const (
 	TenantUsageTableID                  = 45
 	SQLInstancesTableID                 = 46
 	SpanConfigurationsTableID           = 47
+	RoleIDSequenceID                    = 48
```

Comment:
>This feels like a somewhat sad usage of a constant ID. I buy it, but I don't love it. We could do some other things -- if you're interested. One observation is that because of the caching protocol of writes to the users tables, we always will contend when updating any row in that table. That means it's totally okay to contend as part of updating the table.

Yeah I believe it has to be constant since `system.users` depends on it. I don't find it sad really. Also what other things can we do here?

>Another note is that the use of a sequence in the system table is a little scary to me.

Agreed, it is uncharted.

>One possibility is we could use a dummy row in the table or something else illegal to store the allocator row? It's a little hacky to be sure, but it's as or more efficient, won't leak IDs, and saves us a fixed ID. WDYT?

Can you explain more here? Are you talking about in the migration?